src/components/SwipeableView/index.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    children: PropTypes.element.isRequired,
+
+};
+
+function SwipeableView({children}) {
+    return children;
+}
+
+SwipeableView.propTypes = propTypes;
+SwipeableView.displayName = 'SwipeableView';
+export default SwipeableView;
```

Comment:
I think we can be less specific in this file and just do a lazy 

```js
export default ({children}) => children;
```

But we should add some explanation in the comments for why the swipe is not made available in web or Android...

Why is it not available on these platforms (specifically Android)?

Reply:
@marcaaron I put a note in the PR to ask you this question specifically, actually. 

Should I instead make Swipeable available across all platforms while I make ReportView have two separate implementations (for iOS and the second for rest)? I think SwipeableView could be used for Android as well in the future.

Reply:
Ah ok, hmm. No I think if we are going to have the keyboard swipeable on iOS then we should just also make it swipeable on Android to keep things consistent.

Reply:
My reasoning behind having SwipeableView just for iOS was because swipe gesture navigation are more natural to iOS than Android.

Reply:
@marcaaron  The last comment went delayed due to a poor internet connection, sorry. Do you suggest making SwipeableView available in both iOS and Android, then?

Reply:
Yes, I answered above that it should be available on both platforms. 

>swipe gesture navigation are more natural to iOS than Android

Curious to hear more what you mean by this, but as an Android user I do plenty of swiping.

Reply:
@marcaaron It was just an observation based on having Android phones having Back Buttons in-built to navigate between screen vs Swipe being necessary to navigate between screens in iPhone. 

Of course, Swiping is in-built to modern apps more and more and it should be no exception! I am an Android user myself. 😅